base/docs/basedocs.jl

```diff
@@ -59,6 +59,26 @@ See the [manual section about modules](@ref modules) for details.
 """
 kw"export"
 
+"""
+    as
+
+`as` is used as a keyword to rename an identifier brought into scope by
+`import` or `using`, for the purpose of working around name conflicts as
+well as for shortening names.
```

Comment:
```suggestion
`as` is used as a keyword to rename an identifier brought into scope by
`import` or `using`, for the purpose of working around name conflicts as
well as for shortening names.  (Outside of `import` or `using` statements,
`as` is not a keyword and can be used as an ordinary identifier.)
```

Reply:
One quirk is that this is perfectly valid:
```jl
julia> import LinearAlgebra as as

julia> as
LinearAlgebra
```

Reply:
Nice!!!